### ISSUE TITLE
Coping with multiple Firefox profiles

### ISSUE THREAD

**Issue by rrthomas** [johannesjo/linux-window-session-manager#81]
I saw issue #77, and I wondered similarly if there's a way to cope with relaunching multiple Firefox profiles (lwsm of course just finds the same desktop file for each Firefox window). Sorry to be extra demanding but in my case manual editing of the session file won't be good enough, because I'd like to save whatever windows I have open in multiple profiles.

**Comment by johannesjo** [comment]
Hi there! Apart from manually editing the session file there isn't a built in way and I think that it would be way to specific to include something like this into the tool.

**Comment by rrthomas** [comment]
Fair enough! But this does seem to be a version of a more general problem, given the other issue about multi-instance apps.

**Comment by johannesjo** [comment]
I don't disagree, but this is a very tricky problem to solve as there are so many different possible behaviors for each app. As I spent most of my free coding time [on this](https://github.com/johannesjo/super-productivity) it's unlikely that I will ever work on this myself. This being said: PRs are welcome! ;)

**Comment by rrthomas** [comment]
Great stuff, if I can get it to work for me apart from this I might have a go!

**Comment by johannesjo** [comment]
I am curious to see what you come up with!! I admit that I haven't dived into it very deeply, but from what I've seen I would say that this it impossible to find a generic solution. :)
